packages/design-system/src/components/popup/utils/getOffset.js

```diff
@@ -117,5 +117,7 @@ export function getOffset(placement, spacing, anchor, dock, popup) {
     y: offsetY,
     width: anchorRect.width,
     height: anchorRect.height,
+    // We want to know where the popRect container stops before scrolling begins
+    bottom: popupRect?.bottom,
```

Comment:
Before (in the tooltip dynamic placement) the `offset.height` that was returned is for the `anchorRect` this meant that the offset was being calculated based off the anchor and not the popup contents.